pkg/scripts/os_amzn.go

```diff
@@ -44,13 +44,14 @@ echo -n "${yum_proxy}" >> /tmp/yum.conf
 sudo mv /tmp/yum.conf /etc/yum.conf
 
 {{ if .CONFIGURE_REPOSITORIES }}
+LATEST_STABLE=$(curl -s https://cdn.dl.k8s.io/release/stable.txt | sed 's/\.[0-9]*$//')
```

Comment:
Let's not use `cdn.dl.k8s.io` directly, but instead use just `dl.k8s.io`, to be on the safe side.